packages/protocol/contracts/L1/TaikoData.sol

```diff
@@ -135,13 +138,12 @@ library TaikoData {
         bytes32 blobHash; // slot 2
```

Comment:
Dani, if we remove `usingBlob`, we can also remove `blobHash` from Block, move both of them to Metadata.

Reply:
Then when a block is to be proven, the prover will also have to pass in the metadata again then check the hash against blk.metaHash.